src/main/java/domain/PetHotel.java

```diff
@@ -0,0 +1,46 @@
+package domain;
+
+import serenitylabs.tutorials.vetclinic.Pet;
+
+import java.util.*;
+
+import static java.util.Comparator.comparing;
+
+/**
+ * Created by pravyada on 9/20/2016.
+ */
+public class PetHotel  {
+    public static final int MAX_SIZE = 20;
+    private  Collection<Pet> petList = new TreeSet<>(comparing(Pet::getName));
+    private static Queue<Pet> petsInWaitingList = new LinkedList<>();
```

Comment:
The waiting list should not be static, as it is associated with the hotel. In general, be very wary of using static fields anywhere.